src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetFiltersMappingPage.tsx

```diff
@@ -0,0 +1,474 @@
+import Link from '@admin/components/Link';
+import ButtonLink from '@admin/components/ButtonLink';
+import getApiDataSetFilterMappings, {
+  AutoMappedFilter,
+  FilterCandidateWithKey,
+  MappableFilter,
+} from '@admin/pages/release/data//utils/getApiDataSetFilterMappings';
+import ApiDataSetMappableTable from '@admin/pages/release/data/components/ApiDataSetMappableTable';
+import ApiDataSetNewItemsTable from '@admin/pages/release/data/components/ApiDataSetNewItemsTable';
+import ApiDataSetAutoMappedTable from '@admin/pages/release/data/components/ApiDataSetAutoMappedTable';
+import getUnmappedFilterErrors from '@admin/pages/release/data/utils/getUnmappedFilterErrors';
+import ApiDataSetMappableFilterColumnsTable from '@admin/pages/release/data/components/ApiDataSetMappableFilterColumnsTable';
+import ApiDataSetNewFilterColumnsTable from '@admin/pages/release/data/components/ApiDataSetNewFilterColumnsTable';
+import {
+  releaseApiDataSetDetailsRoute,
+  ReleaseDataSetRouteParams,
+} from '@admin/routes/releaseRoutes';
+import apiDataSetVersionQueries from '@admin/queries/apiDataSetVersionQueries';
+import apiDataSetQueries from '@admin/queries/apiDataSetQueries';
+import apiDataSetVersionService, {
+  FilterCandidate,
+  FilterColumnMapping,
+  PendingMappingUpdate,
+} from '@admin/services/apiDataSetVersionService';
+import NotificationBanner from '@common/components/NotificationBanner';
+import Tag from '@common/components/Tag';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import Accordion from '@common/components/Accordion';
+import AccordionSection from '@common/components/AccordionSection';
+import PageNav, { NavItem } from '@common/components/PageNav';
+import { Dictionary } from '@common/types';
+import useDebouncedCallback from '@common/hooks/useDebouncedCallback';
+import { useQuery } from '@tanstack/react-query';
+import React, { useCallback, useEffect, useState } from 'react';
+import { generatePath, useParams } from 'react-router-dom';
+import omit from 'lodash/omit';
+import { useImmer } from 'use-immer';
+
+export default function ReleaseApiDataSetFiltersMappingPage() {
+  const [autoMappedFilterOptions, updateAutoMappedFilterOptions] = useImmer<
+    Dictionary<AutoMappedFilter[]>
+  >({});
+  const [newFilterOptionCandidates, updateNewFilterOptionCandidates] = useImmer<
```

Comment:
Minor, but might be worth simplifying this state to `newFilterOptions`.

Similar could be done for `newFilterColumns` below as well.